cfgov/unprocessed/js/modules/footer-button.js

```diff
@@ -1,26 +1,64 @@
 /* ==========================================================================
    Footer Button: Scroll to Top
+
+   Code copied from the following with minimal modifications :
+
+   - http://stackoverflow.com/questions/21474678/
+     scrolltop-animation-without-jquery
    ========================================================================== */
+
 'use strict';
 
-var $ = require( 'jquery' );
+// Required modules.
+var dataHook = require( './util/data-hook' );
 
-// TODO: Refactor this file to remove jquery dependency.
-//       http://stackoverflow.com/questions/21474678/
-//       scrolltop-animation-without-jquery
 /**
  * Set up event handler for button to scroll to top of page.
  */
 function init() {
-  var duration = 300;
+  var scrollDuration = 300;
+  var behaviorElement = dataHook.get( 'return-to-top' );
 
-  // Disable Google Tag Manager tracking on this link.
-  $( '.js-return-to-top' ).attr( 'data-gtm_ignore', 'true' );
+  if ( behaviorElement === null ||
+       'requestAnimationFrame' in window === false ) {
+    return;
+  }
 
-  $( '.js-return-to-top' ).click( function( event ) {
+  // Disable Google Tag Manager tracking on this link.
+  behaviorElement.setAttribute( 'data-gtm_ignore', 'true' );
```

Comment:
The default behavior of an empty anchor link is still to move the user to the top of the page. If for some reason GTM loaded but this script didn't, we should still ignore the interaction in GTM.